src/Areas/Deploy/Commands/EncodeMermaid.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.IO.Compression;
+using System.Text;
+
+namespace AzureMcp.Areas.Deploy.Commands;
+
+public static class EncodeMermaid
```

Comment:
Add Ref docs to class and method to help explain the purpose and use of this.

Reply:
Done